applications/structure/bar/tests/weak_damping.json

```diff
@@ -0,0 +1,34 @@
+{
+    "General": {
+        "Precision": "double",
+        "Dim": "3",
+        "IsTest": "true"
+    },
+    "SpatialResolution": {
+        "DegreeMin": "1",
+        "DegreeMax": "1",
+        "RefineSpaceMin": "0",
+        "RefineSpaceMax": "0"
+    },
+    "TemporalResolution": {
+        "RefineTimeMin": "0",
+        "RefineTimeMax": "0"
+    },
+    "Application": {
+        "Length": "100.0",
+        "Height": "10.0",
+        "Width": "10.0",
+        "ProblemType": "Unsteady",
+        "WeakDamping": "10000.0",
```

Comment:
note that this is an absurdly high damping coefficient. One would use 0.01 insted. But we (i) test preconditioner performance wrt damping coefficient and (ii) it gives some visible effect using the Neumann BC (= visible lag in the displacement).